app/models/item.js

```diff
@@ -64,6 +85,21 @@ export default cloudinaryUrl.extend({
   imageUrl: Ember.computed.alias("image.imageUrl"),
   designateFullSet: Ember.computed.localStorage(),
 
+  storageTypeName: Ember.computed("storageTypeId", function() {
+    return this.get("storageType.name");
+  }),
+
+  storageTypeIcon: Ember.computed("storageTypeName", function() {
+    switch (this.get("storageTypeName")) {
+      case "Box":
+        return "box-open";
+      case "Pallet":
+        return "pallet";
+      default:
+        return "tag";
+    }
+  }),
+
```

Comment:
We've used this pattern in the past, but the more I see it the more I feel it shouldn't be in the model.
I think a helper is what we should use

```javascript
// storage-icon.js
export default Ember.Helper.helper(function([storageType]) {
  return {
    box: "box-open",
    pallet: "pallet"
  }[storageType.toLowercase()] || "tag"
});
```

and in the `.hbs`

```handlebars
{{fa-icon (storage-icon record.storageType)}
```

Models _should_ be data focused. I've made that mistake myself in the past, I think we should start turning things around